cli.mjs

```diff
@@ -209,14 +267,15 @@ async function studyIDPrompt() {
 }
 
 /** Prompt the user to enter the ID of a participant on the STUDY_ID study */
+// helper to check if the given participant (input) is in firestore under study
```

Comment:
Yeah, it looks like the comment on line 269 is for the function defined on line 277